pom.xml

```diff
@@ -0,0 +1,553 @@
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-site-plugin</artifactId>
+                <version>3.7.1</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <!-- Header on changelog isn't normal -->
+                        <exclude>CHANGES</exclude>
+                        <!-- IDE configuration -->
+                        <exclude>dev/phoenix.importorder</exclude>
+                        <!-- Release L&N -->
+                        <exclude>dev/release_files/LICENSE</exclude>
+                        <exclude>dev/release_files/NOTICE</exclude>
+                        <!-- Exclude data files for examples -->
+                        <exclude>docs/*.csv</exclude>
+                        <exclude>examples/*.csv</exclude>
+                        <!-- Exclude SQL files from rat. Sqlline 1.1.9 doesn't work with
+                             comments on the first line of a file. -->
+                        <exclude>examples/*.sql</exclude>
+                        <exclude>examples/pig/testdata</exclude>
+                        <!-- precommit? -->
+                        <exclude>**/patchprocess/**</exclude>
+                        <!-- Argparse is bundled to work around system Python version
+                             issues, compatibile with ALv2 -->
+                        <exclude>bin/argparse-1.4.0/argparse.py</exclude>
+                        <!-- Not our code -->
+                        <exclude>python/requests-kerberos/**</exclude>
+                        <exclude>python/phoenixdb/phoenixdb/avatica/proto/*</exclude>
+                        <exclude>python/phoenixdb/*.rst</exclude>
+                        <exclude>python/phoenixdb/ci/**</exclude>
+                        <exclude>python/phoenixdb/doc/*.rst</exclude>
+                        <exclude>python/phoenixdb/doc/conf.py</exclude>
+                        <exclude>python/phoenixdb/doc/Makefile</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <!-- Allows us to get the apache-ds bundle artifacts -->
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <inherited>true</inherited>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencyManagement>
+        <dependencies>
+
+            <!-- Intra-project dependencies -->
+            <dependency>
+                <groupId>org.apache.phoenix</groupId>
+                <artifactId>phoenix-core</artifactId>
+                <version>${project.version}</version>
```

Comment:
Add a new phoenix.version to pom.xml, IMO we should depend on a released version of phoenix (not snapshot).

Reply:
Yes, I feel that too. We need to come up about guidelines for how much version variation can come up for compatibility purposes.

Reply:
@joshelser Thoughts here?

Reply:
Agree to use a release version of Phoenix (e.g. last)

I think this goes to the thread up above too -- choosing one version of Phoenix to test against. Largely, PQS should work against all versions of Phoenix, regardless of "internals". Our choice of phoenix-core to use is more about the version we will run unit tests against (which probably also means we can move this dependency to `<scope>test</scope>` too).

Reply:
I didn't quite get your point here @joshelser Can you elaborate?

Reply:
PQS doesn't have a compile-time dependency against Phoenix -- it's only necessary for a Phoenix version when we are building a tarball/packaging for PQS or when we're actually going to run PQS.

I'm thinking that we can make our lives easier by just saying "PQS unit tests run against this version of Phoenix" and running it against different versions is an exercise left to the user.

Reply:
Yes, that makes sense. Refer to my comment for the further plan.